modules/services/model/src/main/java/au/com/shiftyjelly/pocketcasts/models/db/dao/BookmarkDao.kt

```diff
@@ -0,0 +1,25 @@
+package au.com.shiftyjelly.pocketcasts.models.db.dao
+
+import androidx.room.Dao
+import androidx.room.Delete
+import androidx.room.Insert
+import androidx.room.OnConflictStrategy
+import androidx.room.Query
+import androidx.room.Update
+import au.com.shiftyjelly.pocketcasts.models.entity.Bookmark
+
+@Dao
+abstract class BookmarkDao {
+
+    @Insert(onConflict = OnConflictStrategy.REPLACE)
```

Comment:
Now that we're using Room `2.5.1` can we use the new shortcut [@Upsert](https://developer.android.com/jetpack/androidx/releases/room#2.5.0) now?

> Added a new shortcut annotation, @Upsert, which attempts to insert an entity when there is no uniqueness conflict or update the entity if there is a conflict.

Reply:
Good pick up. Do you mind if I add this in a separate PR so we can merge the multiple PRs we have chained?